pkg/remote/deploy_pfe_rbac.go

```diff
@@ -37,7 +37,7 @@ func CreateCodewindRoles(deployOptions *DeployOptions) rbacv1.ClusterRole {
 		rbacv1.PolicyRule{
 			APIGroups: []string{""},
 			Resources: []string{"secrets"},
-			Verbs:     []string{"get", "list", "create", "watch", "delete"},
+			Verbs:     []string{"get", "list", "create", "watch", "delete", "delete", "patch", "update"},
```

Comment:
Is delete duplicated?

Reply:
Good catch, that was an oversight on my part. Fixed!